core/src/main/java/com/netflix/conductor/core/orchestration/ExecutionDAOFacade.java

```diff
@@ -250,6 +250,7 @@ public void removeFromPendingWorkflow(String workflowType, String workflowId) {
      * @param archiveWorkflow if true, the workflow will be archived in the {@link IndexDAO} after removal from  {@link ExecutionDAO}
      */
     public void removeWorkflow(String workflowId, boolean archiveWorkflow) {
+        LOGGER.info("remove workflow {}", workflowId);
```

Comment:
log this after successful removal, we don;t want this log to show up for every API attempt to remove a workflow

Reply:
Removed